packages/snage/src/query/match.ts

```diff
@@ -26,19 +29,31 @@ export const createMatcher = (e: Expression, fields: Field[]): Matcher => {
 
 export const createSimpleExpression = (e: SingleExpression, fields: Field[]): Matcher => {
     const {field, op, value} = e;
-    const conf = fields.find((f) => f.name === field);
+    const conf = fields.find((f) => f.name === field) ?? ImplicitFields.find((f) => f.name === field);
     if (!conf) {
         throw new Error('illegal state');
     }
     if (conf.list) {
         return (note) => {
-            if (!note[field]) {
+            const noteValues = valueByName(field, note);
+            if (!noteValues) {
                 return false;
             }
-            return (note[field] as []).some((noteValue) => checkValue(noteValue, value, conf.type, op));
+            return (noteValues as unknown[]).some((noteValue) => checkValue(noteValue, value, conf.type, op));
         };
     }
-    return (note) => checkValue(note[field], value, conf.type, op);
+    return (note) => checkValue(valueByName(field, note), value, conf.type, op);
+};
+
+const valueByName = (name: string, note: MatcherNote): undefined | unknown => {
+    switch (name) {
+        case ContentField.name:
+            return note.content;
```

Comment:
Alternativ könnte man hier auch überall mit `fields` und `NoteValues` arbeiten, die bereits die implicit fields und values enthalten. Dann würden hier alle Spezialfälle wieder rausfliegen, nehme ich. Genauso in `query/parser.ts`. Man muss eigentlich nur in der API und bei `snage set` darauf achten, dass die impliziten Felder nicht in den NoteValues enthalten sind.
Obwohl, selbst in der API könnte man `content` und `summary` in die values packen, wenn die Namen eh reserviert sind. Dann bräuchte nur die Serialisierung in `set` eine Sonderbehandlung, nehme ich an.